### PR TITLE
Docs: Adds a page for enums

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/docs/pages/guides/database/postgres/enums.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+select * 
+from person 
+where current_mood = 'sad';
+```
+
+## Managing Enums
+
+You can manage your Enums using the `alter type` statement. Here are some examples:
+
+## Updating Enum Values
```
Comment (kevcodez) [inline apps/docs/pages/guides/database/postgres/enums.mdx:69]:
```suggestion
### Updating Enum Values
```

Should be one level lower?

File: apps/docs/pages/guides/database/postgres/enums.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+alter type mood add value 'content';
+```
+
+### Removing Enum Values
+
+You can remove values from an existing Enum using the `ALTER TYPE` statement:
+
+{/* prettier-ignore */}
+```sql
+alter type mood add value 'content';
```
Comment (kevcodez) [inline apps/docs/pages/guides/database/postgres/enums.mdx:97]: Query is for adding a value, but it should be the query to remove

File: apps/docs/pages/guides/database/postgres/enums.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+Check your existing Enum values by querying the enum_range function:
+
+{/* prettier-ignore */}
+```sql
+select enum_range(null::mood);
+```
+
+## Reordering Enum Values
+
+Once an Enum is in use, it's difficult to make changes the the order of the values. However, you can do it by dropping and recreating the Enum. Here's how you can do it for the `mood` Enum above, reodering the values to be in alphabetical order.
```
Comment (kevcodez) [inline apps/docs/pages/guides/database/postgres/enums.mdx:111]:
```suggestion
Once an Enum is in use, it's difficult to make changes to the order of the values. However, you can do it by dropping and recreating the Enum. Here's how you can do it for the `mood` Enum above, reodering the values to be in alphabetical order.
```

File: apps/docs/pages/guides/database/postgres/enums.mdx [hunk 4]
```diff
@@ -0,0 +1,169 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'enums',
+  title: 'Managing Enums in Postgres',
+  description: 'Define a strict set of values for application.',
```
Comment (charislam) [inline apps/docs/pages/guides/database/postgres/enums.mdx:6]:
```suggestion
  description: 'Define a strict set of values that can be used in table and function definitions.',
```

Nit: perhaps more specific?

File: apps/docs/pages/guides/database/postgres/enums.mdx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+{/* prettier-ignore */}
+```sql
+update person
+set current_mood = 'excited'
+where name = 'Alice';
+```
+
+### Adding Enum Values
+
+To add new values to an existing Postgres Enum, you can use the `ALTER TYPE` statement. Here's how you can do it:
```
Comment (charislam) [inline apps/docs/pages/guides/database/postgres/enums.mdx:96]:
```suggestion
To add new values to an existing Postgres Enum, you can use the `ALTER TYPE` statement.
```

**Review by charislam — APPROVED** [review]

**Pull request merged**